packages/cli/src/commands/report.ts

```diff
@@ -60,21 +64,71 @@ Example: npx @salesforce-ux/slds-linter report ${options.directory}
           configPath: normalizedOptions.configEslint
         });
 
-        // Generate report
-        spinner.text = 'Generating SARIF report...';
-        const combinedReportPath = path.join(normalizedOptions.output, 'slds-linter-report.sarif');
-        await ReportGenerator.generateSarifReport([...styleResults, ...componentResults], {
-          outputPath: combinedReportPath,
-          toolName: 'slds-linter',
-          toolVersion: LINTER_CLI_VERSION
-        });
+        // Generate report based on format
+        const reportFormat = options.format.toLowerCase();
+        
+        if (reportFormat === 'sarif') {
+          spinner.text = 'Generating SARIF report...';
+          const combinedReportPath = path.join(normalizedOptions.output, 'slds-linter-report.sarif');
+          await ReportGenerator.generateSarifReport([...styleResults, ...componentResults], {
+            outputPath: combinedReportPath,
+            toolName: 'slds-linter',
+            toolVersion: LINTER_CLI_VERSION
+          });
+          Logger.success(`SARIF report generated: ${combinedReportPath}\n`);
+        } else if (reportFormat === 'csv') {
+          spinner.text = 'Generating CSV report...';
```

Comment:
I think you should move this method to ReportGenerator and create a method in there like generateCSVReport().